docs/developer.rst

```diff
@@ -118,6 +138,9 @@ PostgreSQL and creating the .pgpass file as described below:
   Create .pgpass in your home directory:
   localhost:*:testdb:postgres:Password12!
 
+
+
+
```

Comment:
@ethanwhite, I will remove these lines in a clean up commit after the final review

Reply:
All PR, in other packages depend on these changes to work well